ISM43362/ISM43362.cpp

```diff
@@ -196,6 +196,61 @@ bool ISM43362::dhcp(bool enabled)
     return (_parser.send("C4=%d", enabled ? 1 : 0) && check_response());
 }
 
+/*
+  * @brief  Update modem_country_code.
+  * @param  country_codeP: User's country code.
+  * @retval boolean value : false ->error
+
+WiFi modem is only able to receive 4 country codes: CA, US, FR and JP.
```

Comment:
Here the code mentions 4 country codes, but later we can see a big table with more than that ... do we only support a subset of all available country code ? worth an explanation in the comments

Reply:
I am adding an explanation in the comment.